test/e2e/specs/specs-i18n/wp-gutenboarding-i18n-spec.js

```diff
@@ -0,0 +1,66 @@
+import config from 'config';
+import LanguagePickerComponent from '../../lib/components/gutenboarding-language-picker';
+import * as dataHelper from '../../lib/data-helper.js';
+import * as driverHelper from '../../lib/driver-helper.js';
+import * as driverManager from '../../lib/driver-manager.js';
+import AcquireIntentPage from '../../lib/pages/gutenboarding/acquire-intent-page.js';
+import DesignsPage from '../../lib/pages/gutenboarding/designs-page.js';
+import DomainsPage from '../../lib/pages/gutenboarding/domains-page.js';
+import FeaturesPage from '../../lib/pages/gutenboarding/features-page.js';
+import NewPage from '../../lib/pages/gutenboarding/new-page.js';
+import PlansPage from '../../lib/pages/gutenboarding/plans-page.js';
+import StylePreviewPage from '../../lib/pages/gutenboarding/style-preview-page.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+
+const locale = driverManager.currentLocale();
+
+describe( `Gutenboarding translations @i18n (${ locale })`, function () {
```

Comment:
I like your test writing style! Small and concise steps and clear flow are appreciated.

As an FYI - in the parallel Playwright suite I have written a [set of tests](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/specs-playwright/wp-gutenboarding__language.ts) to switch languages then check that expected text is present. That suite doesn't do as much as your test here (only checks localiztion in `AcquireIntent`) but I plan to expand it to include the cases you're testing here in the future.

Should these tests live under normal Calypso testing or in a separate I18N side of things?

Reply:
> Should these tests live under normal Calypso testing or in a separate I18N side of things?

AFAIK, the I18n tests are provided with different locales env variable and would re-run the tests for every Mag16 locale. Also, failed tests are being reported in a different Slack channel.